test/support/regexes.ts

```diff
@@ -4,6 +4,9 @@ const numberRegex = new RegExp(/[0-9]+/);
 const dateRegex = new RegExp(
   /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z)/,
 );
+const versionRegex = new RegExp(
+  /Currently running version: \d{1,3}.\d{1,3}.\d{1,3}/,
```

Comment:
Is it possible to restrict this to just the version without the text?

Reply:
sure, but then its not that strict any more

Reply:
true, I guess it is finding a balance, I think it is fine for now